frontend/app/(dashboard)/bidding/[project_id]/submit/page.tsx

```
@@ -0,0 +1,242 @@
+  return (
+    <div className="container py-8 mx-auto">
+      <div className="flex flex-col md:flex-row gap-6">
+        {/* Project Details Card */}
+        <div className="md:w-1/2">
+          {project && (
+            <div className="bg-white rounded-lg shadow-lg overflow-hidden border border-gray-100 transition-all hover:shadow-xl">
+              <div className="px-6 py-4 border-b bg-gradient-to-r from-blue-50 to-indigo-50">
+                <h2 className="text-2xl font-bold text-gray-800">{project.title}</h2>
+                <div className="flex items-center gap-4 mt-2 text-sm text-gray-600">
+                  <span className="flex items-center gap-1">
+                    <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 text-green-600" viewBox="0 0 20 20" fill="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)